HBN/README.md

```diff
@@ -29,10 +29,12 @@ Note: Some of the following COINS access instructions were copied from [the HBN
     ```shell
     HBN/
     ├── all_data_dicts_Aug_2018.zip
-    ├── Data Dictionaries/
+    ├── data_dictionaries/
     ├── dictionary.py
     └── README.md
     ```
+Note: That the unziped file should be renamed from `Data Dictionaries/` to
+`data_dictionaries`
```

Comment:
@JessyD I disagree. The spirit of these `dictionary.py` scripts is to take the data as-is and reduce the manual steps necessary to get the BIDS data dictionaries out.